experiments/ClimaEarth/test/component_model_tests/prescr_seaice_tests.jl

```diff
@@ -1,5 +1,6 @@
 import Test: @test, @testset
 import Dates
+import NCDatasets
```

Comment:
Is this needed?

Reply:
Yes, it's needed because we call `TimeVaryingInput` in this file. I guess we had only ever run this test script after including other files that imported NCDatasets, but now this file is self-contained